bundles/persistence/org.openhab.persistence.rrd4j/OSGI-INF/rrd4j.xml

```diff
@@ -9,10 +9,12 @@
     http://www.eclipse.org/legal/epl-v10.html
 
 -->
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.openhab.persistence.rrd4j">
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate" name="org.openhab.persistence.rrd4j">
```

Comment:
since there is no implementation of active and deactivate adding the methods here can be omitted